src/cmd-buildextend-metal

```diff
@@ -63,7 +63,7 @@ if [ $# -ne 0 ]; then
 fi
 
 case "$basearch" in
-    "x86_64"|"aarch64"|"s390x") use_anaconda=;;
+    "x86_64"|"aarch64"|"s390x"|"ppc64le") use_anaconda=;;
```

Comment:
This is all the arches, we can drop the case

Reply:
I would still argue there are other arches(riscv,arm32,...) and would leave it as is until anaconda gets dropped completely.

Reply:
Right, but we should aim to just implement those anaconda-less off the bat.

Reply:
100% agree, after this PR we should just remove all of the Anaconda-related code.

Reply:
I don't disagree, but I think that drop of anaconda is out of the scope for this PR.

Reply:
Yep, agreed - this PR as is is fine.